merge master into live-based 0813

luyajun0205 opened the pull request.

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Merge branch 'live-based' into master

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Merge branch 'master' into master-sync-work

Commit:
Merge pull request #24 from OPS-E2E-Prod/master-sync-work

Resolve syncing conflicts from master-sync-work to master

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

juntakata committed:
token is valid until it goes for 90 days

Refresh Token Max Inactive Time is currently 90 days. it was 14 days in the past. 

https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

luyajun0205 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C13%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201808130708327768-40%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=R%2B7noDJTFYTULJEFtdrCtrwCtJLCzSR%2Bs0AYxksQwrc%3D&st=2018-08-13T07%3A07%3A57Z&se=2018-09-13T07%3A12%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

azuresdkci committed:
Update CLI extensions available doc (#987)

* Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=415217470
https://github.com/Azure/azure-cli-extensions/commit/3af42722afb8d650392212fec3ccccaaf991d41b

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

v-susu-zz commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs-ref-conceptual/azure-cli-extensions-list.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/azure-cli-extensions-list.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-extensions-list?view=azure-cli-latest&branch=pr-en-us-40) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-extensions-list?view=azure-cli-2017-03-09-profile&branch=pr-en-us-40) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C13%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201808131727345144-40%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QrALkggVtow7q2J13S1%2BI2Jv0auYEJJfRHzCdXpHX4M%3D&st=2018-08-13T17%3A26%3A30Z&se=2018-09-13T17%3A31%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

troydai committed:
Update release-notes-azure-cli.md

Commit:
Copyedit release notes

sptramer committed:
Merge pull request #990 from troydai/patch-8

Update release-notes-azure-cli.md for 2.0.44 release

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

v-susu-zz commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs-ref-conceptual/azure-cli-extensions-list.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/azure-cli-extensions-list.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-extensions-list?view=azure-cli-latest&branch=pr-en-us-40) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-extensions-list?view=azure-cli-2017-03-09-profile&branch=pr-en-us-40) (azure-cli-2017-03-09-profile) |
[docs-ref-conceptual/install-azure-cli.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/install-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/install-azure-cli?view=azure-cli-latest&branch=pr-en-us-40) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/install-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-40) (azure-cli-2017-03-09-profile) |
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-40) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-40) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C14%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201808141938258019-40%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=K0OtWeBxxv9AeBb%2Fxn6SBGohuW9QZ8VD7o7c7oZEiaQ%3D&st=2018-08-14T19%3A37%3A27Z&se=2018-09-14T19%3A42%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

troydai committed:
Azure CLI 2.0.45 Release Notes

Commit:
Copyedit release notes.

rloutlaw committed:
Merge pull request #1003 from troydai/patch-8

Azure CLI 2.0.45 Release Notes

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

azuresdkci committed:
Update CLI extensions available doc (#1007)

* Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=422787136
https://github.com/Azure/azure-cli-extensions/commit/2c423f0f5e32ea21e618f3717b5cacfa5035142f

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

sptramer committed:
Merge pull request #980 from juntakata/master

Token is valid until it goes for 90 days

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)
